### ISSUE TITLE
Question:  what's the difference among `OPENDAPNOFITY=..` and `NOTIFYUSER=...`  and `statusNotify=...` 

### ISSUE THREAD

**Issue by wwlwpd** [StormSurgeLive/asgs#666]
@jasonfleming - I am wondering if we can collapse these down to 1; at the very least I'd like to avoid any other _notification_ type lists. Any thoughts?

**Comment by jasonfleming** [comment]
Yes its a mess. It has grown by accretion over time. The NOTIFYUSER list goes way back to when we first started, and would send emails when a new advisory had been detected, etc. It is now completely vestigial (as far as I know) and can be removed.

OPENDAPNOTIFY is still in use, it sends emails when new results have been posted to an OPeNDAP server. It is still needed for CERA, at least.

statusNotify is something I added for notifying people upon ASGS startup that json status messages are going to be posted. However, I don't actually use it, and it can also be removed.

**Comment by wwlwpd** [comment]
Ah great, thank you. Let me think about the next step - we'll see where those things are outside of just config files (if they're anywhere - probably not based on what you said).

**Comment by wwlwpd** [comment]
Actually, I'll close this. :)

**Comment by jasonfleming** [comment]
Reopening this to address a couple related parameters with the hope that we can reduce, streamline, and disambiguate this. Looking at the code, we also have `EMAILNOTIFY` (list where we can send an email when a new advisory is issued, or ASGS detects that a run has failed) as well as `ASGSADMIN` (single email address of the ASGS Operator that goes into queue scripts so that Operator gets email from the HPC system when it detects that a job has failed) and `NOTIFY_SCRIPT` which contains the contents of the emails that are being sent. Unlike `NOTIFYUSER`, these parameters are actually used, although their utility is probably marginal, and overlapping to some extent. Going forward, we need some kind of heads up when there is a failure, but that can be a Slack message, text, phone call, etc. We are trying to get away from needing a results notifier, especially one separate from `OPENDAPNOTIFY`, and definitely don't need to be notified when a new advisory is issued. How can we cut down, streamline, and generalize this?
